addons/map/functions/fnc_blueForceTrackingUpdate.sqf

```diff
@@ -26,6 +26,28 @@ if (GVAR(BFT_Enabled) and {(!isNil "ACE_player") and {alive ACE_player}}) then {
         };
     };
 
+    if (GVAR(BFT_ShowPlayerNames)) then {
+        _playersToDrawMarkers = allPlayers select {side _x == _playerSide};
+
+        {
+            private _markerType = [_x] call EFUNC(common,getMarkerType);
+            private _colour = format ["Color%1", side _x];
+
+            private _marker = createMarkerLocal [format ["ACE_BFT_%1", _forEachIndex], [(getPos leader _x) select 0, (getPos leader _x) select 1]];
+            _marker setMarkerTypeLocal _markerType;
+            _marker setMarkerColorLocal _colour;
+            _marker setMarkerTextLocal (name _x);
+
+            GVAR(BFT_markers) pushBack _marker;
+        } forEach _playersToDrawMarkers;
+
+        _groupsToDrawMarkers = _groupsToDrawMarkers select {
```

Comment:
I filter players out of the _groupsToDrawMarkers array when BFT_ShowPlayerNames is true in case BFT_HideAiGroups is false, this way you can have players with individual names and icons and AI groups shown normally. 

It probably would be best to wrap that part in a condition that checks whether or not BFT_HideAiGroups is false though.